System/Primitive.cpp

```diff
@@ -158,6 +158,102 @@ namespace RTE {
 		}
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void RoundBoxPrimitive::Draw(BITMAP *drawScreen, Vector targetPos) {
```

Comment:
This is pretty unimportant, but I'd say it'd be better to rename this to `RoundedBoxPrimitive`